content/docs/alerting/configuration.md

```diff
@@ -214,9 +214,7 @@ source_match_re:
 
 Receiver is a named configuration of one or more notification integrations.
 
-__Other receiver implementations available in version 0.0.4 of Alertmanager
-are not implemented yet. We are gladly accepting any contributions to add them
-to the new implementation.__
+__We're not actively adding notifiers at the moment as we try to figure out something more flexible. For the time being, we recommend implementing custom notifiers via the webhook notifier.__
```

Comment:
I'm not sure how much it's about being on the search for something more flexible, the jsonnet approach would definitely be more flexible but I also see it being more error prone and can easily get out of sync as users would need to update those themselves separate from an Alertmanager update.

I think this is primarily because we can't take on the additional maintenance burden, therefore we chose to support only the major providers out of the box, and additional need to be added through the webhook integration.

My 2 cents, but happy to hear what others have to say.

Reply:
My thoughts are in line with @brancz

Reply:
cool by me

Reply:
Wonder if the best action is just to remove this section of the documentation. What do you think?

Reply:
I'd leave it given the frequency of requests, but remove the "as we try to figure out something more flexible. For the time being,"